### PR TITLE
chore: fix the lint script on package to resolve

### PR DESCRIPTION
**Pull request by hkang1**
## Description

PROBLEM:
`npm run lint` doesn't seem to resolve and hangs at the `npm run stylelint` command. The seems like `npm run eslint` and `npm run stylelint` are scripts designed specifically for `Makefile` targets to be able to pass arguments to them. `npm run lint` however should be a self contained run instead.

FIX:
Update `npm run lint` to use `npm run lint:eslint` and `npm run lint:style` instead (which has fixed file targets to lint)

## Test Plan

- [ ] run `npm run lint` locally and it should complete without issues (instead of hanging at `npm run stylelint`)

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: webui/react/package.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
     "prettier": "prettier",
     "eslint": "eslint",
     "stylelint": "stylelint --config .stylelintrc.js",
-    "lint:eslint": "eslint \"src/**/*.{js,jsx,ts,tsx}\"",
-    "lint:style": "stylelint --config .stylelintrc.js \"src/**/*.{css,less,scss}\"",
```
Comment (hamidzr) [inline webui/react/package.json:18]: was the --config here redundant?

**Review by hamidzr — APPROVED** [review]
lgtm assuming `make -C webui check` with and without `-j` work as before.

File: webui/react/package.json (hunk 1 above)
Comment (hkang1) [reply webui/react/package.json:18]: huh, looks like it. I'll clean that up.
```
❯ npx stylelint --config .stylelintrc.js "src/**/*.{css,less,scss}"

src/pages/F_ExpList/glide-table/MultiSortMenu.module.scss
 16:3  ✖  Expected align-items to come before display  order/properties-alphabetical-order

1 problem (1 error, 0 warnings)

❯ npx stylelint "src/**/*.{css,less,scss}"

src/pages/F_ExpList/glide-table/MultiSortMenu.module.scss
 16:3  ✖  Expected align-items to come before display  order/properties-alphabetical-order

1 problem (1 error, 0 warnings)
```

**Comment by hkang1** [comment]
> lgtm assuming `make -C webui check` with and without `-j` work as before.

ah I didn't check the `-j` option but just gave that a shot and it looks good. thanks

**Pull request merged**